plugins/iommu/fu-plugin-iommu.c

```diff
@@ -48,6 +48,7 @@ fu_plugin_iommu_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 
 	if (priv == NULL || !priv->has_iommu) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_FW);
```

Comment:
This should be CONFIG_OS

Reply:
Ahh, on the logic that it's in the blocklist?

Reply:
Yeah, you can compile the kernel without it, you can blacklist it, you can put it in soft mode.  All those things can cause this outcome.
Also - this points out to me that I don't think the check that we have for an IOMMU being present is a good enough test.  We probably should be making sure that it's actually being used for translation.  But that's a whole separate can of worms because I'm not sure the kernel exports enough on it right now.